src/vtkF3DAlembicImporter.cxx

```diff
@@ -0,0 +1,351 @@
+#include "vtkF3DAlembicImporter.h"
+
+#include <vtkActor.h>
+#include <vtkActorCollection.h>
+#include <vtkCamera.h>
+#include <vtkDoubleArray.h>
+#include <vtkFloatArray.h>
+#include <vtkImageData.h>
+#include <vtkImageReader2.h>
+#include <vtkImageReader2Factory.h>
+#include <vtkLight.h>
+#include <vtkMatrix4x4.h>
+#include <vtkObjectFactory.h>
+#include <vtkPointData.h>
+#include <vtkPoints.h>
+#include <vtkPolyData.h>
+#include <vtkPolyDataMapper.h>
+#include <vtkProperty.h>
+#include <vtkQuaternion.h>
+#include <vtkRenderer.h>
+#include <vtkShaderProperty.h>
+#include <vtkSmartPointer.h>
+#include <vtkStringArray.h>
+#include <vtkTexture.h>
+#include <vtkTransform.h>
+#include <vtkUniforms.h>
+#include <vtkUnsignedShortArray.h>
+
+#include <vtksys/SystemTools.hxx>
+
+#include <Alembic/AbcGeom/All.h>
+// #include <Alembic/AbcCoreHDF5/All.h> Not sure if we want to support legacy HDF5 stuff
```

Comment:
HDF5 library is included in VTK in case you decide to support it.

Reply:
I will leave it there and gather feedback from users to ascertain their use case before making a final decision.

Reply:
ok, then remove the comented line in the meantime

Reply:
Done